### ISSUE TITLE
dotLottie will not load on web

### ISSUE THREAD

**Issue by arasrezaei** [lottie-react-native/lottie-react-native#1190]
### Description
Image.resolveAssetSource is not defined on web.
https://github.com/lottie-react-native/lottie-react-native/blob/ad223dd0cd8417a4455465952f2c52431b1e0f2b/packages/core/src/LottieView/utils.ts#L31
for resolving the .lottie path i fixed it with expo-asset.

Version: 6.6.0

**Comment by matinzd** [comment]
For web, you need to follow the instructions in the README: 
https://github.com/lottie-react-native/lottie-react-native?tab=readme-ov-file#web

**Comment by arasrezaei** [comment]
> For web, you need to follow the instructions in the README: https://github.com/lottie-react-native/lottie-react-native?tab=readme-ov-file#web

Yes I followed instructions, and it plays json files, and remote .lottie files, only when you require local .lottie file (no problem with .json) this error arises

**Comment by matinzd** [comment]
Can you provide a minimal reproducable?

**Comment by ravanscafi** [comment]
I am having the same issue. Metro is configured to accept `.lottie` assets. It works on native, but not on web. If I switch to `json`, it starts working again.
Probably it's related to this issue: https://github.com/necolas/react-native-web/issues/1666

**Comment by BoavistaLudwig** [comment]
I am having the same issue right now. @ravanscafi @arasrezaei have you found a fix?

**Comment by ravanscafi** [comment]
@BoavistaLudwig not really, the workaround is using a remote Lottie file or going back to JSON.
